app/bundles/CampaignBundle/Config/config.php

```diff
@@ -174,6 +174,9 @@
             ],
             'mautic.campaignrange.type.action' => [
                 'class' => 'Mautic\CampaignBundle\Form\Type\EventType',
+                /** CAPTIVEA.CORE START **/
+                'arguments' => 'mautic.factory',
```

Comment:
Use directly the objects you need. If you need entity manager there, use `'arguments' => 'doctrine.orm.entity_manager',`. I don't know what the problem with forms is.